docs/flutterbloccoreconcepts.md

```diff
@@ -192,6 +192,45 @@ MultiBlocListener(
 )
 ```
 
+### BlocConsumer
+
+**BlocConsumer** exposes a `builder` and `listener` in order react to new states. `BlocConsumer` is analogous to a nested `BlocListener` and `BlocBuilder` but reduces the amount of boilerplate needed. `BlocConsumer` should only be used when it is necessary to both rebuild UI and execute other reactions to state changes in the `bloc`. `BlocConsumer` takes a required `BlocWidgetBuilder` and `BlocWidgetListener` and an optional `bloc`, `BlocBuilderCondition`, and `BlocListenerCondition`.
+
+If the `bloc` parameter is omitted, `BlocConsumer` will automatically perform a lookup using
```

Comment:
Could you add more detail about `perform a lookup` for what? I think it is `perform a lookup for the BlocA instance`.